test/CMakeLists.txt

```diff
@@ -0,0 +1,51 @@
+set(SRCROOT "${PROJECT_SOURCE_DIR}/test/src")
+
+include_directories("${PROJECT_SOURCE_DIR}/include")
+include_directories("${PROJECT_SOURCE_DIR}/extlibs/headers")
+
+# System is always built
+SET(SYSTEM_SRC
+    "${SRCROOT}/Vector2.cpp"
+    "${SRCROOT}/Vector3.cpp"
+    "${SRCROOT}/TestUtilities/System.hpp"
+    "${SRCROOT}/TestUtilities/System.cpp"
+)
+SET(SYSTEM_LIB "sfml-system")
+
+if(SFML_BUILD_WINDOW)
+    SET(WINDOW_SRC
+        "${SRCROOT}/Rect.cpp"
```

Comment:
Shouldn't Rect be in the graphics tests?

Reply:
Woops, forgot to move it after I created the graphics section. 🙂